src/components/buttons/send-recap-email-button.vue

```diff
@@ -37,10 +38,19 @@ export default {
     }
   },
   methods: {
+    sendEvent() {
+      this.sendEventToMatomo(
+        EventCategories.GENERAL,
+        "Recap",
```

Comment:
Y a-t-til des contre indications à mettre des noms d'event explicites ? 

```suggestion
        "Clic : bouton d'accès au récapitulatif d'email",
```

Reply:
Non pas vraiment, tu as raison, merci beaucoup !